docs/site-example/site.conf

```diff
@@ -35,6 +35,14 @@
     -- Wireless channel.
     channel = 1,
 
+    -- List of supported wifi rates (optional, implies basic_rate)
```

Comment:
"implies basic_rate" is a bit unclear. Maybe add "required if supported_rates" is set to the "basic_rate" comment?